.github/workflows/build_and_test.yml

```diff
@@ -28,3 +28,28 @@ jobs:
         
       - name: Test
         run: dotnet test -f ${{ matrix.framework }} --no-build --no-restore
+  
+  test-documentation:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: |
+            6.0.x
+            7.0.x
+      
+      # used for documentation
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.5'
```

Comment:
@dtchepak do we want to update it?

Reply:
I think i'm locally on 2.6.5 at the moment. 😅 Happy to update to the latest version that doesn't break anything 😊

Reply:
I tried to update it but it looks like we need to update bundler and packages locally first. 
I'm merging as is. Let's do it in a separate PR.